hasher-matcher-actioner/hmalib/indexers/s3_indexers.py

```diff
@@ -54,6 +57,16 @@ def load(cls, bucket_name: str):
             )
             return pickle.loads(index_file_bytes)
 
+    @classmethod
+    def get_oldest_last_modified(cls, bucket_name: str) -> datetime:
+        """
+        Get the update time of the oldest index.
+        """
+        objects = get_s3_client().list_objects_v2(
+            Bucket=bucket_name, Prefix=cls.INDEXES_PREFIX
+        )
+        return max(map(lambda item: item["LastModified"], objects["Contents"]))
```

Comment:
How is "LastModified" stored? depending on the timestamp counting up wouldn't this actually be the most recently modified?

Reply:
Yes. This is the most recently modified timestamp.

I am not accounting for a case where one index is built but the other is not.